src/itemviewformaction.cpp

```diff
@@ -129,7 +128,6 @@ void ItemViewFormAction::prepare()
 		}
 
 		f->modify("article", "replace_inner", formatted_text);
-		f->set("articleoffset", "0");
```

Comment:
I don't understand why this line was removed. (To be fair, I also don't understand why it was added in the first place.) Can you explain why you deleted it?

Reply:
Sure, that line places the user at the top of the article, so whenever an update is performed and this function is called you get moved from where you were reading. Removing this line stops this from happening.

This should not affect the article on first load since this variable is already set in _stfl/itemview.sftl_. And if i had to guess it's being set here because someone did not know it was being set there (or vice versa).

Reply:
This line was introduced in 805a249b simultaneously with the variable in stfl markup, so I'm pretty sure the author was aware of it being set in both places. However, the code back than was much simpler: itemview didn't exist as a separate entity, and it was run "in one go" without any state. I don't understand why the author reset `articleoffset` there; it doesn't make sense to me.

Let's continue this discussion in #387.